src/confdb/confdb.c

```diff
@@ -1310,9 +1310,15 @@ static int confdb_get_domain_internal(struct confdb_ctx *cdb,
         }
     } else {
         /* default */
-        domain->case_sensitive = true;
-        domain->case_preserve = true;
+        if (strcasecmp(domain->provider, "ad") == 0) {
```

Comment:
Yes, there was a copy-paste error, one of those was supposed to be case_preserve.

Without this patch the domain context is wrongly initialized if the domain is AD (it defaults to case sensitive). The following sssctl patch would not work because it would use case sensitive search even for AD. I found this with gdb when looking into the values in the domain context.